ruby/vampires.rb

```diff
@@ -0,0 +1,47 @@
+puts "How many employees will be processed?"
+employee_number = gets.chomp
+n = employee_number.to_i
+n.times do
+  puts "What's your name?"
+  employee_name = gets.chomp
+  puts "How old are you? What year were you born? (right/ wrong)"
```

Comment:
So for the age we should do some calculations based off the age a user submits.  For example, we could do something like...

```puts "How old are you?"
age_input = gets.chomp.to_i
puts "When were you born?"
birth_year_input = gets.chomp.to_i
correct_age = 2017 - age_input == birth_year_input```